src/main/resources/application.yml

```diff
@@ -2,7 +2,7 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:5432/betting
     username: postgres
-    password: admin
+    password: 210902
```

Comment:
Revert this to initial value and try to not commit your passwords (let me know when you get to this and I'll show you a trick that helps with this)